src/applications/edu-benefits/5490/containers/IntroductionPage.jsx

```diff
@@ -104,6 +105,7 @@ export class IntroductionPage extends React.Component {
         </div>
         <SaveInProgressIntro
           buttonOnly
+          downtime={this.props.route.formConfig.downtime}
```

Comment:
It would be cool if this was a default prop instead, so that we don't have to worry about future or WIP forms adding this prop.